moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -48,6 +48,8 @@
 
 #include "ui_motion_planning_rviz_plugin_frame.h"
 
+#include "string"
```

Comment:
This should probably be `#include <string>` for consistency, and should go above all of the moveit includes.